### ISSUE TITLE
Browsing behind a proxy doesn't work

### ISSUE THREAD

**Issue by danielfranca** [liri-browser/liri-browser#94]
Mac OS X 10.10
Trying to browse behind a proxy doesn't work.
Even if  I set http_proxy and https_proxy env vars.

**Comment by M4SSD35TRUCT10N** [comment]
@danielfranca Must be specific to Mac OS X 10.10 since I don't have any problems with that under Windows 7.

**Comment by danielfranca** [comment]
@M4SSD35TRUCT10N That's possible, I'll test it under Linux/Windows later.

**Comment by danielfranca** [comment]
Maybe it's related to this thing? https://forum.qt.io/topic/58225/webengineview-and-network-proxy
Solution: Wait until Qt5.6? =|

**Comment by danielfranca** [comment]
Maybe this information can ring the bell for someone :)
After call QNetworkProxyFactory::setUseSystemConfiguration in main.cpp I can see the "new tab" recent visited thumbs and the omni auto suggestion works (but very slow)... but I still can't browse to any website.

**Comment by danielfranca** [comment]
It seems that the web view loader is never ready :(

**Comment by tim-sueberkrueb** [comment]
Yeah, looks like we'll have to wait for Qt 5.6. But it will be released in December, so it's not that long away.
